charts/dealer/templates/dealer-deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "dealer.fullname" . }}
+  labels:
+    app: {{ template "dealer.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "dealer.fullname" . }}
+      release: {{ .Release.Name }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: {{ template "dealer.fullname" . }}
+        release: "{{ .Release.Name }}"
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "3333"
+        prometheus.io/scrape: "true"
+      {{- with .Values.annotations }}
+        {{ toYaml . | nindent 8 }}
+      {{- end }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}@{{ .Values.image.digest }}"
+        env:
+        - name: NETWORK
+          value: regtest
+        - name: LOG_LEVEL
+          value: debug
+        - name: ACTIVE_STRATEGY
+          value: OKEX_PERPETUAL_SWAP
+        - name: ACTIVE_WALLET
+          value: REMOTE_WALLET
+        - name: OKEX5_KEY
+          value: key
+        - name: OKEX5_SECRET
+          value: secret
```

Comment:
This can't be right or?

Reply:
all OKEX5_* env. var. are dependent on the exchange account being used and are real secrets so the .envrc just had indication of those to get filled in